modules/setting/mailer.go

```diff
@@ -93,6 +96,21 @@ func newMailService() {
 	MailService.FromName = parsed.Name
 	MailService.FromEmail = parsed.Address
 
+	switch MailService.EnvelopeFrom {
+	case "":
+		MailService.OverrideEnvelopeFrom = false
+	case "<>":
```

Comment:
Why we need this magic?

Reply:
I proposed it. Before, we need two setting options like `USE_DIFFERENT_ENVELOPE_FROM=true` `ENVELOPE_FROM=` to present an empty address.

In RFC, `<>` is the empty return path, and it doesn't cause ambiguity here. Now we can only one setting to decide:
* user wants use the default return-path: empty string (Go default value for a string)
* user wants to use an empty return-path: `<>` as RFC
* user wants to use a special return-path: `a@b.com`

If a user need to set this option, they must be very familiar with the email system, and `<>` is very clear to them.